adapters/dmx/dmx.go

```diff
@@ -186,6 +189,9 @@ func (adapter *DmxAdapter) MakeRequests(request *openrtb.BidRequest, req *adapte
 	}
 
 	dmxReq.Imp = imps
+	if anyHasId == false {
```

Comment:
Nitpick: Please consider adding a line break before this condition for readability.